packages/amplication-client/src/Layout/PageContent.scss

```diff
@@ -33,20 +31,16 @@ $tabs-width: 300px;
     @include scrollbars(10px, var(--black60), transparent);
     flex: 1;
     overflow: auto;
-    height: 100%;
-    flex: 1;
-    padding: var(--large-spacing);
     position: relative;
     box-sizing: border-box;
-    height: 100%;
 
     @media only screen and (max-width: $page-content-min-width) {
       min-width: calc(#{$page-content-min-width} - var(--menu-width));
     }
   }
+
   .log-container {
     flex: 1;
-    width: 50%;
```

Comment:
why did you remove this? is that related to the issue?

Reply:
It's a clean up. Not related specifically to this issue.